hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/filtermanagement/TargetFilterTargetGrid.java

```diff
@@ -83,4 +86,9 @@ public void addColumns() {
 
         getColumns().forEach(column -> column.setHidable(true));
     }
+
+    private Column<ProxyTarget, String> addControllerIdColumn() {
+        return GridComponentBuilder.addColumn(this, ProxyTarget::getControllerId).setId(TARGET_CONTROLLER_ID)
```

Comment:
In my opinion it is better to just extend the GridComponentBuilder with #addControllerIdColumn in order not to duplicate it three times for each grid. It is kind of a special case, that ideally should not be part of GridComponentBuilder, however due to the fact there are multiple target grids used in different places it appears to be the most suitable solution

Reply:
Had the same considerations, since the builder only creates columns that each type has, I did not add the controllerId to the builder. But to avoid duplicated code, I will move it into the `GridComponentBuilder`.